oneandone/resource_oneandone_server.go

```diff
@@ -33,17 +33,22 @@ func resourceOneandOneServer() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"fixedsize": {
```

Comment:
To keep it consistent with the rest of the resource why don't we use `fixed_instance_size` and also allow for name/part of the name or fixed instance size ID to be provided. 
Also it would be beneficial if we had a [data source](https://www.terraform.io/docs/configuration/data-sources.html) `fixed_instance_size`